src/components/containers/flex.ts

```diff
@@ -23,6 +23,8 @@ type FlexType = {
     tablet_ai?: string
     tablet_jc?: string
     tablet_fw?: string
+    scroll?: boolean | string
+    margin?: string
```

Comment:
It seems like they are unused and can be removed, Would you please check 🙏

Reply:
They are used in the file src/templates/article.tsx 
In these components: 
**<StickyBreadCrumbsWrapper scroll={visible}>**
** <Flex fd="column" margin="0 auto" ai="center">**